tcp_check/README.md

```diff
@@ -39,6 +40,15 @@ Configuration Options
 
 [Restart the Agent][6] to start sending TCP service checks and response times to Datadog.
 
+#### Containerized
+For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying the parameters below.
+
+| Parameter            | Value                                                                                            |
+|----------------------|--------------------------------------------------------------------------------------------------|
+| `<INTEGRATION_NAME>` | `tcp_check`                                                                                      |
+| `<INIT_CONFIG>`      | blank or `{}`                                                                                    |
+| `<INSTANCE_CONFIG>`  | `{"name": "<TCP_CHECK_INSTANCE_NAME>", "host":"%%host%%", "port":"<CONTAINER_PORT>"}` |
```

Comment:
Do we want to use `%%port%%` here or let people figure out with the doc?

Reply:
Updated 😄